.travis.yml

```diff
@@ -30,7 +30,7 @@ matrix:
 before_script:
   - travis_retry composer self-update
   - travis_retry composer config --global github-oauth.github.com "$GITHUB_TOKEN"
-  - travis_retry composer global require "fxp/composer-asset-plugin:~1.1.3"
+  - travis_retry composer global require "fxp/composer-asset-plugin"
```

Comment:
@kenny-wealth Can you please specify a version?

Reply:
Updated